src/meta/app_env_validator.cpp

```diff
@@ -46,6 +46,27 @@ bool check_slow_query(const std::string &env_value, std::string &hint_message)
     return true;
 }
 
+bool check_deny_client(const std::string &env_value, std::string &hint_message)
+{
+    std::vector<std::string> sub_sargs;
+    utils::split_args(env_value.c_str(), sub_sargs, '*', true);
+
+    std::string invalid_hint_message = "Invalid deny client args, valid include: reject*all, "
+                                       "reject*write, reject*read; delay*all, delay*write, "
+                                       "delay*read";
```

Comment:
I notice that the `delay` policy is not really delay, it is not return any response to client, both of them are reject, the difference is whether return error code to client to trigger query meta. I recommend not using delay and rename it.

Reply:
`return` no resp will cause `ERR_TIMEOUT`, it is like `delay some seconds until timeout` for client side, so I use it and no find other name